modules/gcp_helpers/assets_feed/main.tf

```diff
@@ -1,13 +1,14 @@
 data "google_projects" "all" {
-  filter = "parent.id=${var.organization}"
+  filter = var.project_filter
```

Comment:
Which projects are returned when project_filter is empty? All available projects? please verify

Reply:
all available projects are returned when project filter is empty